content/presubmission_inquiry.md

```diff
@@ -1,29 +1,28 @@
-Dear Dr. Markel:
+Dear Dr. Markel,
 
-I am writing to submit a Ten Simple Rules article entitled “Ten Simple Rules for
-Deep Learning in Biology.” Deep learning (DL) is exploding in popularity and is
-increasingly finding its way into biological data analysis. However, since most
-biologists receive little (or no) formal data science training, using deep learning
-properly remains a daunting task.
+it is with great enthusiasm that I am writing to submit a presubmission inquiry for a Ten Simple Rules article entitled “Ten Simple Rules for Deep Learning in Biology.” Deep learning (DL) is currenty exploding in popularity and is
+increasingly finding its way into biological data analysis given its ability to detect high-dimensional, and for humans invisible, patterns in large-scale biological datasets. However, since most
+biologists and also many data scientists have received and still receive little (or no) formal training in DL, correct application of DL to biological data analysis remains a daunting task hindering DL-accelerated scientific progress and enhanced biological insight.
```

Comment:
Too wordy, IMO. Would revise to: "biologists rarely receive formal training in DL and its correct application to biological data analysis remains a daunting task that hinders scientific progress and enhanced biological insight."